.github/workflows/ci.yml

```diff
@@ -47,7 +47,9 @@ jobs:
       - run: yarn install --frozen-lockfile
 
       - run: yarn shadow-cljs release npm
+      - run: yarn run bundle-ts
       - run: yarn shadow-cljs compile test
-      
+
       - run: node out/node-tests.js
       - run: yarn jest js/tests
+      - run: yarn tsd
```

Comment:
Do you think it makes sense to run tsd tests with jest so all the js tests stay together? https://github.com/jest-community/jest-runner-tsd

Reply:
sure I can look into it. The tsd tests were much harder than expected to get running so if this doesn't require me to do a ton of work to figure out how to run them again, I'm down.

Reply:
yeah, not a big deal so if it doesn't just work don't worry about it. Just update the test:js script

Reply:
@smothers just looked into this and it only has 2 stars which isn't super reassuring. Would rather not deal with the added complexity of wrapping tsd, which already has awful debugging, in another test runner. But down to discuss.

Reply:
ok, don't worry about it